Cargo.toml

```diff
@@ -3,7 +3,9 @@ members = [
     "programs/*",
     "insurance-engine",
     "snapshot-parser",
-    "merkle-tree"
+    "merkle-tree",
+    "cli",
```

Comment:
still a bit generic - wdyt @ochaloup ?

Reply:
Agree. It could be some more descriptive name. There is already an TS CLI for admin operations under `/packages`.

This name could be something with `bot` or `ci` or `db`, maybe?